standalone-metastore/metastore-common/src/main/java/org/apache/hadoop/hive/metastore/conf/MetastoreConf.java

```diff
@@ -559,6 +559,8 @@ public static ConfVars getMetaConf(String name) {
     DELEGATION_TOKEN_STORE_CLS("metastore.cluster.delegation.token.store.class",
         "hive.cluster.delegation.token.store.class", METASTORE_DELEGATION_MANAGER_CLASS,
         "Class to store delegation tokens"),
+    DESCTABLE_ENABLE_PARTITION_STATS("desctable.enable.partitionstats", "desctable.enable.partitionstats", true, 
```

Comment:
This config is used by HS2 and so move it to HiveConf.java. Also, use the name "hive.describe.table.ignore.stats" and keep the default as false.

Reply:
Moved to HiveConf.java. Should it be called "hive.describe.**partitioned**table.ignore.stats" since the change only affects partitioned tables?

Reply:
yeah, fine with partitioned table.